celestia-app/minimal/genesis.json

```diff
@@ -0,0 +1,295 @@
+{
+  "genesis_time": "2022-05-11T21:04:33.756553272Z",
+  "chain_id": "ephemeral",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "22020096",
+      "max_gas": "-1",
+      "time_iota_ms": "1000"
+    },
+    "evidence": {
+      "max_age_num_blocks": "100000",
+      "max_age_duration": "172800000000000",
+      "max_bytes": "1048576"
+    },
+    "validator": {
+      "pub_key_types": [
+        "ed25519"
+      ]
+    },
+    "version": {}
+  },
+  "app_hash": "",
+  "app_state": {
+    "auth": {
+      "params": {
+        "max_memo_characters": "256",
+        "tx_sig_limit": "7",
+        "tx_size_cost_per_byte": "10",
+        "sig_verify_cost_ed25519": "590",
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "celes1z4t2fnem459xcfj5kknnd5yy8rp4frhdnnkel9",
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        }
+      ]
+    },
+    "bank": {
+      "params": {
+        "send_enabled": [],
+        "default_send_enabled": true
+      },
+      "balances": [
+        {
+          "address": "celes1z4t2fnem459xcfj5kknnd5yy8rp4frhdnnkel9",
+          "coins": [
+            {
+              "denom": "celes",
+              "amount": "800000000000"
+            }
+          ]
+        }
+      ],
```

Comment:
Maybe better use uceles for denom? 🧐

Reply:
I see `uceles` and `CELES` as the denoms in the genesis.json.
```
"denom_metadata": [
        {
          "description": "The native staking token of the Celestia network.",
          "denom_units": [
            {
              "denom": "uceles",
              "exponent": 0,
              "aliases": [
                "microceles"
              ]
            },
            {
              "denom": "CELES",
              "exponent": 6,
              "aliases": []
            }
          ],
          "base": "uceles",
          "display": "CELES",
          "name": "CELES",
          "symbol": "CELES"
        }
      ]
```
It looks like they're case insensitive ([relevant PR](https://github.com/cosmos/cosmos-sdk/pull/3092))

Since we're using a big number I'll swap the initial amount to `uceles`.